view/js/main.js

```diff
@@ -620,6 +620,12 @@ function liveUpdate(src) {
 		in_progress = false;
 		update_item = 0;
 
+		if (data.length < 100) {
```

Comment:
This is not ideal since it will break if we change anything to the templates you've also changed. Since we are dealing with  HTML snippet, please load it in a jQuery element and use DOM traversal functions to ensure there's no items in the response.

Reply:
This would be far beyond my capabilities. I was happy finding the `length` function.:-)

Reply:
@MrPetovan like I said: I tried some stuff with JS but failed miserably. If the system should check for some HTML stuff, then I need clear advice how to do it.

Reply:
You can transform a HTML string in a DOM object like this: `$(<HTML string>)`. Then you can figure out if there's a post element with the regular jQuery DOM traversal function `.find()` on the jQuery object you created.